src/app/cic/fields.js

```diff
@@ -99,6 +99,7 @@ module.exports = {
     validate: [
       "required", "date",
       {type: "before", arguments: [new Date().toISOString().split("T")[0]]},
+      {type: "after", arguments: ["1899-01-31"]}
```

Comment:
just wondering if this needs to be "1899-12-31"?

Reply:
You're right, nice catch - will fix